Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,442 @@
+# Migrating from Analytics mobile extension to Edge Network using the EdgeBridge extension
+## Overview
+This hands-on tutorial provides end-to-end instructions on how to migrate to Edge Network from Analytics to EdgeBridge mobile extension.
+
+There are other beginning states the customer can be in like using ACP Analytics, v4 extension, etc. These usages can potentially be migrated to AEP and from there, this tutorial can be applied afterwards. However, this should be determined on a case-by-case basis, as for example in the v4 extension case, it may be more effective to simply implement AEP extensions itself without the need for Edge Bridge migration path.
+
+Tutorial setup:
+
+Initial test app that is based on prerequisites (implements Analytics)
+Final test app after all the tutorial steps have been implemented (implements EdgeBridge)
+
+Tutorial steps:
```

Comment:
Do you think this doc is too long?  We can have TOC.

 Or we can separate the sections to several pages and link from the main page?